scio-bigquery/src/test/scala/com/spotify/scio/bigquery/types/TypeProviderTest.scala

```diff
@@ -74,6 +78,10 @@ class TypeProviderTest extends FlatSpec with Matchers {
     S4.tableDescription shouldBe "Table S4"
   }
 
+  it should "be serializable" in {
+    SerializableUtils.ensureSerializable(S1(1))
+  }
```

Comment:
Is this the correct place for this test? 
Is this the right way to test?
Beam is running this before converting, so I thought it would be nice to use this.

Reply:
Please wait. I don't think I am doing this correctly here.